webtau-core/src/main/java/com/twosigma/webtau/data/table/header/Header.java

```diff
@@ -111,6 +111,16 @@ public String toString() {
         return namesByIndex.toString();
     }
 
+    /**
+     * column index by column name
+     * @param columnName column name to get index for
+     * @return column index or -1 if not found
+     */
+    public int findColumnIdxByName(String columnName) {
+        Integer idx = indexByName.get(columnName);
```

Comment:
out of curiosity: what does indexByName.get(null) return? null or exception?

Reply:
null